src/clients/SpokePoolClient.ts

```diff
@@ -954,7 +797,7 @@ export class SpokePoolClient extends BaseAbstractClient {
     return this.oldestTime;
   }
 
-  async findDepositV3(depositId: number, destinationChainId: number, depositor: string): Promise<V3DepositWithBlock> {
+  async findDeposit(depositId: number, destinationChainId: number, depositor: string): Promise<DepositWithBlock> {
```

Comment:
Should this be a V3 deposit

Reply:
This change removes all V2 types, so everything is implicitly a V3 type anyway :+1: